### ISSUE TITLE
Force white background in HTML

### ISSUE THREAD

**Issue by akissinger** [zxcalc/book#5]
In “dark mode” on Safari/iOS, the HTML background is black. This looks ok for text and math, but figures are mostly invisible. Setting the background to white manually should fix this.

**Comment by jvdwetering** [comment]
Yeah this should have the easy fix of adding `background-color: white;` to the CSS of `.svg-diagram`

**Comment by jvdwetering** [comment]
This should now be fixed in latest version.
